pkgs/development/python-modules/fiona/default.nix

```diff
@@ -35,6 +35,7 @@ buildPythonPackage rec {
   checkInputs = [
     pytest
     boto3
+    mock
```

Comment:
looking at the setup.py, `mock` should be only [installed on python<3.4](https://github.com/Toblerity/Fiona/blob/1.8.6/setup.py#L286) i.e.
```
    ++ stdenv.lib.optionals (pythonOlder "3.4") [ mock ];
```
besides that, LGTM

Reply:
Done

Reply:
thanks!